vite.config.ts

```diff
@@ -0,0 +1,42 @@
+import { defineConfig } from 'vitest/config'
+import react from '@vitejs/plugin-react'
+
+
+export default defineConfig({
+  plugins: [react()],
+  base: '',
+  resolve: {
+    alias: {
+      api: '/src/api',
+      app: '/src/app',
+      components: '/src/components',
+      "redux-store": '/src/redux-store',
+      sources: '/src/sources',
+      tests: '/src/tests',
+      webroot: '/src/webroot'
+    }
+  },
+  server: {
+    port: 3000,
+    proxy: {
+      '^/api/.*': 'https://dev.dissco.tech'
```

Comment:
Do we need to change this if we deploy? Or is this only used locally?

Reply:
No its for development, its the proxy to the API, already removed it from the PR